--- includes/common.inc Sat Jun  1 19:01:12 2002
+++ includes/common.inc Fri Jun 14 17:18:14 2002
@@ -146,7 +146,7 @@
   $result = db_query("SELECT * FROM variable");
   while ($variable = db_fetch_object($result)) {
     if (!isset($conf[$variable->name])) {
-      $conf[$variable->name] = $variable->value;
+      $conf[$variable->name] = unserialize($variable->value);
     }
   }
 
@@ -163,7 +163,7 @@
   global $conf;
 
   db_query("DELETE FROM variable WHERE name = '%s'", $name);
-  db_query("INSERT INTO variable (name, value) VALUES ('%s', '%s')", $name, $value);
+  db_query("INSERT INTO variable (name, value) VALUES ('%s', '%s')", $name, serialize($value));
 
   $conf[$name] = $value;
 }
